src/manageiq_client/api.py

```diff
@@ -230,6 +240,13 @@ def reload_if_needed(self):
         if self._data is None:
             self.reload()
 
+    def query_string(self, **params):
+        """Specify query string to use with the collection.
```

Comment:
e.g. `search = api.collections.automate.query_string(depth="-1", search_options="state_machines")` for accessing "/api/automate?depth="-1"&search_options="state_machines"